packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -155,3 +158,119 @@ export function getNetworkName(
+  rollup: '0x5eF0D09d1E6204141B4d37530808eD19f60FBa35',
+  classicOutboxes: {
+    '0x667e23ABd27E623c11d4CC00ca3EC4d0bD63337a': 0,
+    '0x760723CD2e632826c38Fef8CD438A4CC7E7E1A40': 30
+  }
+}
+
+export const networksList: { [chainId: number]: L1Network | L2Network } = {
+  [ChainId.ArbitrumOne]: {
+    chainID: 42161,
+    name: 'Arbitrum One',
+    explorerUrl: 'https://arbiscan.io',
+    partnerChainID: 1,
+    isArbitrum: true,
+    tokenBridge: mainnetTokenBridge,
+    ethBridge: mainnetETHBridge,
+    confirmPeriodBlocks: 45818,
+    isCustom: false,
+    retryableLifetimeSeconds: SEVEN_DAYS_IN_SECONDS,
+    nitroGenesisBlock: 22207817,
+    /**
+     * Finalisation on mainnet can be up to 2 epochs = 64 blocks on mainnet
+     */
+    depositTimeout: 888000
+  },
+  [ChainId.ArbitrumNova]: {
+    chainID: 42170,
+    confirmPeriodBlocks: 45818,
+    ethBridge: {
+      bridge: '0xc1ebd02f738644983b6c4b2d440b8e77dde276bd',
+      inbox: '0xc4448b71118c9071bcb9734a0eac55d18a153949',
+      outbox: '0xD4B80C3D7240325D18E645B49e6535A3Bf95cc58',
+      rollup: '0xfb209827c58283535b744575e11953dcc4bead88',
+      sequencerInbox: '0x211e1c4c7f1bf5351ac850ed10fd68cffcf6c21b'
+    },
+    explorerUrl: 'https://nova-explorer.arbitrum.io',
+    isArbitrum: true,
+    isCustom: false,
+    name: 'Arbitrum Nova',
+    partnerChainID: 1,
+    retryableLifetimeSeconds: SEVEN_DAYS_IN_SECONDS,
+    tokenBridge: {
+      l1CustomGateway: '0x23122da8C581AA7E0d07A36Ff1f16F799650232f',
+      l1ERC20Gateway: '0xB2535b988dcE19f9D71dfB22dB6da744aCac21bf',
+      l1GatewayRouter: '0xC840838Bc438d73C16c2f8b22D2Ce3669963cD48',
+      l1MultiCall: '0x8896d23afea159a5e9b72c9eb3dc4e2684a38ea3',
+      l1ProxyAdmin: '0xa8f7DdEd54a726eB873E98bFF2C95ABF2d03e560',
+      l1Weth: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+      l1WethGateway: '0xE4E2121b479017955Be0b175305B35f312330BaE',
+      l2CustomGateway: '0xbf544970E6BD77b21C6492C281AB60d0770451F4',
+      l2ERC20Gateway: '0xcF9bAb7e53DDe48A6DC4f286CB14e05298799257',
+      l2GatewayRouter: '0x21903d3F8176b1a0c17E953Cd896610Be9fFDFa8',
+      l2Multicall: '0x5e1eE626420A354BbC9a95FeA1BAd4492e3bcB86',
+      l2ProxyAdmin: '0xada790b026097BfB36a5ed696859b97a96CEd92C',
+      l2Weth: '0x722E8BdD2ce80A4422E880164f2079488e115365',
+      l2WethGateway: '0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD'
+    },
+    nitroGenesisBlock: 0,
+    depositTimeout: 888000
+  },
+  [ChainId.Mainnet]: {
+    chainID: ChainId.Mainnet,
+    name: 'Ethereum Mainnet',
+    explorerUrl: 'https://etherscan.io',
+    partnerChainIDs: [42161, 42170],
+    blockTime: 14,
+    isCustom: false,
+    isArbitrum: false
+  }
+}
+
+export const networksListArray = Object.keys(networksList)
+  .map((chainId: string) => networksList[Number(chainId)])
+  .sort((a, b) => (a.isArbitrum > b.isArbitrum ? -1 : 1)) // sort l2 networks together, before l1
+
+export const networkStyleMap: {
+  [chainId: number]: { img: string; btnThemeClass: string }
+} = {
```

Comment:
@spsjvc Is this association of chain-id to their stylings required? If not this, are we okay with if-else conditions in the NetworkSelectionContainer component?